packages/components/src/ForumScreen.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Rename ForumScreen to ForumPage and put is under components/src/pages

Reply:
done